pkg/crc/preflight/labels.go

```diff
@@ -13,7 +13,8 @@ const (
 	NetworkMode
 
 	// Keep it last
-	lastLabelName // will be used in OS-specific go files to extend LabelName
+	// will be used in OS-specific go files to extend LabelName
+	lastLabelName // nolint
```

Comment:
@anjannath why it is require `nolint` and also this looks like comment change.

Reply:
This is unused when building for windows, so `golangci-lint` complains also `// nolint` has to be in its own line, hence the shifting around of the previous comment :)